chainio/elcontracts/writer.go

```diff
@@ -176,14 +178,15 @@ func (w *ELChainWriter) RegisterBLSPublicKey(
 	ctx context.Context,
 	blsKeyPair *bls.KeyPair,
 	operator types.Operator,
+	blsPubkeyCompendiumAddr common.Address,
```

Comment:
we don't need to pass it , we are already initializing in ELContractsClient - you can just call `GetBLSPublicKeyCompendiumContractAddress` - https://github.com/Layr-Labs/eigensdk-go/blob/master/chainio/clients/el_contracts_client.go#L68

Reply:
addressed in [aa70c37](https://github.com/Layr-Labs/eigensdk-go/pull/47/commits/aa70c37f9a6305568dbf4b90a56a44ad2bdcca7b)